src/handler.cjs

```diff
@@ -32,4 +33,9 @@ if (extractorEnv) {
   }
 }
 
-exports.handler = datadog(loadSync(taskRootEnv, handlerEnv), { traceExtractor });
+try {
+  exports.handler = datadog(loadSync(taskRootEnv, handlerEnv), { traceExtractor });
+} catch (error) {
+  emitTelemetryOnErrorOutsideHandler(error, handlerEnv, Date.now());
```

Comment:
I might be wrong but is this `fire and forget` way of invoking the async method? I'm just wondering if we need to add `.then()` or `.error()` to make sure it ends?

Reply:
thanks! yes I think you're right, added